sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/arm/models/implementation/IndependentChildImpl.java

```diff
@@ -68,6 +68,10 @@ public String resourceGroupName() {
         }
     }
 
+    protected String parentName() {
```

Comment:
What error message associates with this?

Reply:
The error message is we could not expose a protected / public variable that hasn't been used. So I add this function to use the variable.